src/Language/Haskell/Liquid/Liquid.hs

```diff
@@ -232,21 +233,24 @@ dumpCs cgi = do
 pprintMany :: (PPrint a) => [a] -> Doc
 pprintMany xs = vcat [ F.pprint x $+$ text " " | x <- xs ]
 
-instance Show Cinfo where
-  show = show . F.toFix
-
 solveCs :: Config -> FilePath -> CGInfo -> GhcInfo -> Maybe [String] -> IO (Output Doc)
 solveCs cfg tgt cgi info names = do
   finfo            <- cgInfoFInfo info cgi
-  F.Result r sol _ <- solve (fixConfig tgt cfg) finfo
+  let fcfg          = fixConfig tgt cfg
+  F.Result r sol _ <- solve fcfg finfo
   let resErr        = applySolution sol . cinfoError . snd <$> r
   -- resModel_        <- fmap (e2u cfg sol) <$> getModels info cfg resErr
   let resModel_     = e2u cfg sol <$> resErr
-  let resModel      = resModel_  `addErrors` (e2u cfg sol <$> logErrors cgi)
+  let lErrors       = applySolution sol <$> logErrors cgi
```

Comment:
@pbougou By comparison to the old code, we should not `applySolution` to the `logErrors`; any reason why we do?